docker-compose.yml

```diff
@@ -2,7 +2,7 @@ services:
   biblioteca:
     image: ghcr.io/biblioverse/biblioteca-docker:2.0.2
     build:
-      target: prod
+      target: dev
```

Comment:
I would keep the prod target to avoid Breaking Changes.

Reply:
I did that change on purpose; we use the docker-compose mainly for dev, so it just makes sense to use the dev target.
We can switch to the prod target if we want to make sure the prod image works.

If we don't use the `dev` target for ... dev ... then I don't understand why we need it at all.

Reply:
We have 2 choices I think:
1. always make sure the docker compose file for the documentation is correctly aligned with this one when we make changes
2. the docker-compose file in the root foolder is the one we reference in the doc and we create a docker-compose.dev.override.yml for local dev 

I would go with option 2

Reply:
I don't get those arguments.
https://biblioverse.github.io/biblioteca/installing/1-initial-setup/
The documentation points to a totally different docker-compose.

This means that we tailor this docker-compose only for dev and I would prefer if it used the dev image for that.

Reply:
What I mean is that everytime something changes in docker compose, like container, mariadb or typesense version, we need to change it in 2 different files and if we use this one as the one for documentation and an override for dev then we are sure the doc is always up to date with the root docker compose which is just easier to avoid forgetting to maintain. But I can totally live with it

Reply:
Personally I use the dev image also on "prod", as it's easier for Kobo Sync debugging..
I have nothing against having 2 docker-compose file to clarify this issue.

Reply:
>  like container, mariadb or typesense version
Renovate handles that already, but it's true that we could forget updating some elements when changing them.

Reply:
let's trust your instinct and switch to `dev`